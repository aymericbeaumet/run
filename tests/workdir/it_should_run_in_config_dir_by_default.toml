[[run]]
name = "ls"
cmd = ["bash", "-c", "ls ./fixtures"]