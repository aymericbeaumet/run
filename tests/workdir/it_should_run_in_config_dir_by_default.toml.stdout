[ls] a
[ls] b
[ls] c